File: day1/part1.js
// :^) console.log(`Day 1 | Part 1 - Solution: ${eval(require("fs").readFileSync("input.txt").toString())}`);

const { readFileSync } = require("fs");
const inpArr = readFileSync("input.txt").toString().split('\n');
const answer = inpArr.reduce((acc, curr) => acc + parseInt(curr), 0);
console.log(`Day 1 | Part 1 - Solution: ${answer}`);

File: day1/part1.java
import java.util.Scanner;
import java.io.File;
import java.io.FileNotFoundException;

class part1 {

    public static void main(String[] args) throws FileNotFoundException {
        Scanner input = new Scanner(new File("input.txt"));
        int total = 0;
        while(input.hasNextLine()) {
            total += Integer.parseInt(input.nextLine());
        }
        input.close();
        System.out.println("Day 1 | Part 1 - Solution: " + total);
    }

}